### PR TITLE
API doc: Correct Base URL and Dataset URL.

### PR DESCRIPTION
**Pull request by cyhasuka**
The correct URL is http://<host_address>/v1/api , not http://<host_address>/api/v1 .

### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [x] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

### CODE REVIEW

File: docs/references/ragflow_api.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ RAGFlow offers RESTful APIs for you to integrate its capabilities into third-par
...
 ## Base URL
 ```
-http://<host_address>/api/v1/
+http://<host_address>/v1/api/
 ```
 
 ## Dataset URL
 ```
-http://<host_address>/api/v1/dataset
+http://<host_address>/v1/api/dataset
```
Comment (KevinHuSh) [inline docs/references/ragflow_api.md]:
```suggestion
http://<host_address>/api/v1/dataset
```

This is designed to be.

File: docs/references/ragflow_api.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ RAGFlow offers RESTful APIs for you to integrate its capabilities into third-par
 
 ## Base URL
 ```
-http://<host_address>/api/v1/
+http://<host_address>/v1/api/
```
Comment (KevinHuSh) [inline docs/references/ragflow_api.md:11]: Your correction is right.

Comment (cyhasuka) [reply docs/references/ragflow_api.md:11]: Can I understand that in future releases the dataset URL will change to http://<host_address>/api/v1/dataset? Why doesn't it stay synchronized with the base URL?

Comment (KevinHuSh) [reply docs/references/ragflow_api.md:11]: Yes. Because we want to be consistent with others.

**Pull request merged**